src/java/picard/cmdline/CommandLineParser.java

```diff
@@ -340,6 +344,22 @@ public void usage(final PrintStream stream, final boolean printCommon) {
         }
     }
 
+    private void checkForNonASCII(String documentationText, String location) {
+        if (!CharMatcher.ASCII.matchesAllOf(documentationText)) {
+            throw new AssertionError("Non-ASCII character used in documentation of "+callerOptions.getClass().getSimpleName()+" ("+location+"). Only ASCII characters are allowed.");
+        }
+    }
+
+    private String convertFromHtml(final String textToConvert) {
+
+        String regex1 = "<br />|<p>|</p>|<table>|</table >|<a href='(.*)'>(.*)</a>|<h4>|<pre>|</pre>|<ul>|</ul>|</li>|</tr>|<hr />";
+        String regex2 = "</h4>|<tbody>|</tbody>|<tr>|<th>";
+        String regex3 = "<li>";
+        String regex4 = "</th>";
+
+        return textToConvert.replaceAll(regex1, "\n").replaceAll(regex2, "").replaceAll(regex3, " - ").replaceAll(regex4, "\t");
+    }
```

Comment:
why not just make an array/list of regexes, and then iterate through them all and `replaceAll`?  It seems easier if we want to add more later.